### ISSUE TITLE
get amount of space used/free in the plasma_store

### ISSUE THREAD

**Issue by russellromney** [russellromney/brain-plasma#6]
use `sys.getsizeof(x)` for each named object to calculate the exact size of all the things brain knows

```
brain.size()
# >>> 50000000

brain.used()
# >>> 25000000

brain.free() # just take brain.size() - brain.used()
# >>> 25000000
```

**Comment by russellromney** [comment]
Need to use buffer size as described here https://arrow.apache.org/docs/python/memory.html

**Comment by russellromney** [comment]
Fixed by #14

**Comment by russellromney** [comment]
> Need to use buffer size as described here https://arrow.apache.org/docs/python/memory.html

Actually used the built-in `data_size` property of each `brain.client.list()` item.
